robottelo/datafactory.py

```diff
@@ -133,6 +150,31 @@ def valid_data_list():
 
 
 @datacheck
+def valid_emails_list():
+    """Returns a list of valid emails."""
+    return [
+        u'{0}@example.com'.format(gen_string('alpha')),
+        u'{0}@example.com'.format(gen_string('alphanumeric')),
+        u'{0}@example.com'.format(gen_string('numeric')),
+        u'{0}@example.com'.format(gen_string('alphanumeric', 48)),
+        u'{0}+{1}@example.com'.format(gen_alphanumeric(), gen_alphanumeric()),
+        u'{0}.{1}@example.com'.format(gen_alphanumeric(), gen_alphanumeric()),
```

Comment:
I removed the usage of gen_alphanumeric() and used gen_string instead.  Also this is the only place where gen_alphanumeric() is used so I have removed that from import too :)